database/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM mysql:8.0.29
+
+ENV MYSQL_DATABASE veryrezsi
+
+# All scripts in docker-entrypoint-initdb.d/ are automatically
+# executed during container startup
+COPY ./scripts/ /docker-entrypoint-initdb.d/
```

Comment:
The law requiring a new line at the end of a file extends to both mortal and divine realms.